### PR TITLE
fix(NcAppDetailsToggle): Rtl support

### PR DESCRIPTION
**Pull request by hamza221**
### ☑️ Resolves

- part of https://github.com/nextcloud/mail/issues/10227

### 🖼️ Screenshots

🏚️ Before | 🏡 After
---|---
B | A

### 🚧 Tasks

- [x] ...

### 🏁 Checklist

- [x] ⛑️ Tests are included or are not applicable
- [x] 📘 Component documentation has been extended, updated or is not applicable
- [x] 3️⃣ Backport to `next` requested with a Vue 3 upgrade

### CODE REVIEW

File: src/components/NcAppContent/NcAppDetailsToggle.vue [hunk 1]
```diff
@@ -46,6 +51,9 @@ export default {
 		title() {
 			return t('Go back to the list')
 		},
+		isRTL() {
+			return isRTL(getLanguage())
```
Comment (DorraJaouad) [inline src/components/NcAppContent/NcAppDetailsToggle.vue:55]: BTW, `isRTL` already has a fallback `getLanguage()`
https://github.com/nextcloud-libraries/nextcloud-l10n/blob/03d07c211f7381e927955ce9fe899cdd5ccc2a26/lib/locale.ts#L33-L34

**Review by ShGKme — APPROVED** [review]

File: src/components/NcAppContent/NcAppDetailsToggle.vue (hunk 1 above)
Comment (susnux) [reply src/components/NcAppContent/NcAppDetailsToggle.vue:55]:
```suggestion
			return isRTL()
```

Comment (susnux) [reply src/components/NcAppContent/NcAppDetailsToggle.vue:55]: But this does not depend on any runtime variable, so please move this to `setup` instead @hamza221

File: src/components/NcAppContent/NcAppDetailsToggle.vue [hunk 2]
```diff
@@ -35,6 +39,7 @@ export default {
 	components: {
 		NcButton,
 		ArrowRight,
+		ArrowLeft,
 	},
 	setup() {
 		return {
```
Comment (susnux) [inline src/components/NcAppContent/NcAppDetailsToggle.vue:45]:
```suggestion
		return {
			isRTL: isRTL(),
```

**Review by susnux — APPROVED** [review]

**Pull request merged**